tests/formats_test/selinux_test.py

```diff
@@ -92,6 +94,31 @@ def test_mounting_xfs(self):
         an_fs.unmount()
         os.rmdir(mountpoint)
 
+    @patch("blivet.util.mount", return_value=0)
+    @patch.object(fs.FS, "_pre_setup", return_value=True)
+    @patch("os.access", return_value=True)
+    # pylint: disable=unused-argument
+    # pylint: disable=no-self-use
+    def test_mount_selinux_context(self, *args):
```

Comment:
Pylint disables needed:
-  `*args` cannot be ommitted, because it is used by patch decorators
- no `self` call triggers "This should be a function" warning, but the method has to be part of the class to be run.